docs/user/ppl/cmd/lookup.rst

```diff
@@ -0,0 +1,376 @@
+	  }
+	}'
+
+Result set::
+
+    {
+      "acknowledged": true,
+      "persistent": {
+        "plugins": {
+          "calcite": {
+            "enabled": "true"
+          }
+        }
+      },
+      "transient": {}
+    }
+
+
+Usage
+=====
+
+Lookup::
+
+    source = table1 | lookup table2 id
+    source = table1 | lookup table2 id, name
+    source = table1 | lookup table2 id as cid, name
+    source = table1 | lookup table2 id as cid, name replace dept as department
+    source = table1 | lookup table2 id as cid, name replace dept as department, city as location
+    source = table1 | lookup table2 id as cid, name append dept as department
+    source = table1 | lookup table2 id as cid, name append dept as department, city as location
+
+
+Example 1: replace
+==================
+
+PPL query::
+
+	>> curl -H 'Content-Type: application/json' -X POST localhost:9200/_plugins/_ppl -d '{
+	  "query" : """
+	  source = worker
+	  | LOOKUP work_information uid AS id REPLACE department
+	  | fields id, name, occupation, country, salary, department
+	  """
+	}'
+
+Result set::
+
+    {
+      "schema": [
+        {
+          "name": "id",
+          "type": "integer"
+        },
+        {
+          "name": "name",
+          "type": "string"
+        },
+        {
+          "name": "occupation",
+          "type": "string"
+        },
+        {
+          "name": "country",
+          "type": "string"
+        },
+        {
+          "name": "salary",
+          "type": "integer"
+        },
+        {
+          "name": "department",
+          "type": "string"
+        }
+      ],
+      "datarows": [
+        [
+          1000,
+          "Jake",
+          "Engineer",
+          "England",
+          100000,
+          "IT"
+        ],
+        [
+          1001,
+          "Hello",
+          "Artist",
+          "USA",
+          70000,
+          null
+        ],
+        [
+          1002,
+          "John",
+          "Doctor",
+          "Canada",
+          120000,
+          "DATA"
+        ],
+        [
+          1003,
+          "David",
+          "Doctor",
+          null,
+          120000,
+          "HR"
+        ],
+        [
+          1004,
+          "David",
+          null,
+          "Canada",
+          0,
+          null
+        ],
+        [
+          1005,
+          "Jane",
+          "Scientist",
+          "Canada",
+          90000,
+          "DATA"
+        ]
+      ],
+      "total": 6,
+      "size": 6
+    }
+
+Example 2: append
+=================
+
+PPL query::
+
+	>> curl -H 'Content-Type: application/json' -X POST localhost:9200/_plugins/_ppl -d '{
+	  "query" : """
+	  source = worker
+	  | LOOKUP work_information uid AS id APPEND department
+	  | fields id, name, occupation, country, salary, department
+	  """
+	}'
+
+
+Example 3: no inputField
+========================
+
+PPL query::
+
+	>> curl -H 'Content-Type: application/json' -X POST localhost:9200/_plugins/_ppl -d '{
```

Comment:
in docTest, we can use PPL without curl, which is much clearner in doc, for instantce, https://github.com/opensearch-project/sql/blob/main/docs/user/ppl/cmd/fields.rst

Reply:
Calcite is not enabled by default in main branch. Query started with `OS> ` will fail. That's why I use curl formatting.